core/qt-ble.cpp

```diff
@@ -294,10 +295,21 @@ static const struct uuid_match skip_characteristics[] = {
 	{ NULL, }
 };
 
+static const struct uuid_match skip_characteristics_read[] = {
```

Comment:
This can probably be made a bit easier to read and maintain by combining the three `skip_...` lists into one and using a bitmask to encode which characteristic(s) it should be masked for.

Reply:
That could result in more complex code instead of a simplification. The `uuid_match` data structure and the `match_uuid_list` function are not only used for characteristics, but also used for services and those don't have read/write properties.